src/wiktextract/extractor/en/page.py

```diff
@@ -20,56 +20,48 @@
 )
 
 from mediawiki_langcodes import get_all_names, name_to_code
-from wikitextprocessor import NodeKind, WikiNode
 from wikitextprocessor.core import TemplateArgs, TemplateFnCallable
-from wikitextprocessor.parser import LEVEL_KIND_FLAGS, GeneralNode, TemplateNode
+from wikitextprocessor.parser import (
+    LEVEL_KIND_FLAGS,
+    GeneralNode,
+    NodeKind,
+    TemplateNode,
+    WikiNode,
+)
 
-from wiktextract.clean import clean_template_args, clean_value
-from wiktextract.datautils import (
+from ...clean import clean_template_args, clean_value
+from ...datautils import (
     data_append,
     data_extend,
     ns_title_prefix_tuple,
 )
-from wiktextract.form_descriptions import (
-    classify_desc,
-    decode_tags,
-    distw,
-    parse_alt_or_inflection_of,
-    parse_sense_qualifier,
-    parse_word_head,
-)
-from wiktextract.inflection import TableContext, parse_inflection_section
-from wiktextract.linkages import parse_linkage_item_text
-from wiktextract.page import (
+from ...page import (
     LEVEL_KINDS,
     clean_node,
     is_panel_template,
     recursively_extract,
 )
-from wiktextract.parts_of_speech import PARTS_OF_SPEECH
-from wiktextract.tags import valid_tags
-from wiktextract.translations import parse_translation_item_text
-from wiktextract.type_utils import (
-    DescendantData,
-    ExampleData,
-    FormData,
-    LinkageData,
-    SenseData,
-    SoundData,
-    TemplateData,
-    WordData,
-)
-from wiktextract.wxr_context import WiktextractContext
-from wiktextract.wxr_logging import logger
-
+from ...wxr_context import WiktextractContext
+from ...wxr_logging import logger
 from ..ruby import extract_ruby, parse_ruby
 from ..share import strip_nodes
 from .example import extract_example_list_item, extract_template_zh_x
+from .form_descriptions import (
+    classify_desc,
+    decode_tags,
```

Comment:
We should make a generic version of `decode_tags` at some point (at the moment it's couple too much with en stuff), `classify_desc` unfortunately can't be made generic.

`distw` could be at the top level, it just tells the Levenshtein distance between two words (how similar they are), so should be applicable to anything that the `Levenshtein` package supports, which should be at minimum be most languages with Latin script or maybe all scripts.

Reply:
This is not very important, just for the future.

Reply:
I don't know if we really want a complex function for all editions...